src/Databases/PostgreSQL/DatabasePostgreSQL.cpp

```diff
@@ -212,7 +213,7 @@ StoragePtr DatabasePostgreSQL::fetchTable(const String & table_name, ContextPtr
         auto storage = std::make_shared<StoragePostgreSQL>(
                 StorageID(database_name, table_name), pool, table_name,
                 ColumnsDescription{columns_info->columns}, ConstraintsDescription{}, String{},
-                context_, configuration.schema, configuration.on_conflict);
+                context_, configuration.schema, configuration.on_conflict, PostgreSQLSettings{});
```

Comment:
Let's allow `CREATE DATABASE ... ENGINE=PostgreSQL SETTINGS <postgresql_table_settings>`? Just load them the same way as for table engine with `loadFromQuery` in `registerDatabasePostgreSQL` below and pass through the constructor of `DatabasePostgreSQL` and then use to create postgresql tables.